src/Components/Layouts/Footer/Footer.css

```diff
@@ -31,4 +31,8 @@ footer div div div:nth-child(3) a {
 
 #Licence p {
     margin-top: 0;
+}
+
+footer div div div a {
```

Comment:
this is weird and depends too much on order of elements. just add class to that a, or somee of those divs.